src/main/kotlin/app/revanced/patches/ticktick/misc/themeunlock/fingerprints/ProThemesFingerprint.kt

```diff
@@ -0,0 +1,15 @@
+package app.revanced.patches.ticktick.misc.themeunlock.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.ticktick.misc.themeunlock.annotations.UnlockThemesCompatibility
+
+@Name("set-theme")
+@UnlockThemesCompatibility
+@Version("0.0.1")
+object ProThemesFingerprint : MethodFingerprint(
```

Comment:
Make sure you also rename the file
```suggestion
@Name("set-theme-fingerprint")
@UnlockThemesCompatibility
@Version("0.0.1")
object SetThemeFingerprint : MethodFingerprint(
```